File: RightslineDemoApplication_Java/RightslineDemoAppJavaV3/src/main/java/com/rightsline/ConfigSetup.java
package com.rightsline;

import com.google.gson.JsonObject;
import com.google.gson.JsonParser;

import java.io.File;
import java.io.FileReader;
import java.util.HashMap;

public class ConfigSetup {
    public static HashMap<String, String> credentials = new HashMap<>();

    public static boolean getConfigFile() {
        File file = new File("./Config/dev_config.json");
        try {
            JsonObject configJson = new JsonParser().parse(new FileReader(file.getPath())).getAsJsonObject();
            String accessKey = configJson.get("accessKey").getAsString();
            String secretKey = configJson.get("secretKey").getAsString();
            String xApiKey = configJson.get("xApiKey").getAsString();

            credentials.put("accessKey", accessKey);
            credentials.put("secretKey", secretKey);
            credentials.put("xApiKey", xApiKey);

        } catch (Exception e) {
            System.out.println("Please ensure that you have a valid config file in the Config folder ");
            System.out.println("EXCEPTION: " + e);
        }
		return !credentials.isEmpty();
    }

    public static HashMap<String, String> getCredentials() {
        return credentials;
    }
}

File: RightslineDemoAppDotNet/RightslineDemoAppDotNetV3/Config/ConfigSetup.cs
using Newtonsoft.Json.Linq;
using System;
using System.IO;

namespace RightslineDemoAppDotNetV3.Config
{
    public static class ConfigSetup
    {        
        public static string AccessKey { get; set; }
        public static string SecretKey { get; set; }
        public static string ApiKey { get; set; }
        /// <summary>
        /// This method sets up the credentials for the V3 API by reading a config.json file located in the Config folder
        /// </summary>
        public static void GetConfigFile()
        {            
            var folderPath = AppContext.BaseDirectory.Substring(0, AppContext.BaseDirectory.IndexOf("bin", StringComparison.Ordinal));
            var filePath = Path.Combine(folderPath, "Config\\config.json");
            try
            {
                var credentials = JObject.Parse(File.ReadAllText(filePath));
                AccessKey = credentials.GetValue("accessKey").ToString();
                SecretKey = credentials.GetValue("secretKey").ToString();
                ApiKey = credentials.GetValue("xApiKey").ToString();
            }
            catch(Exception e)
            {
                Console.WriteLine(e.Message);
                Console.WriteLine("Please ensure that you have a valid config.json file in the " + folderPath + "Config folder.");
            }          
        }
    }
}
